test/integration/test-amp-inabox.js

```diff
@@ -90,6 +90,7 @@ describe
         const queries = parseQueryString(req.url.substr('/bar'.length));
         expect(queries['cid']).to.equal('');
         expect(queries['sourceUrl']).be.ok;
+        expect(queries['cookie']).to.equal('');
```

Comment:
add a comment that explain where the cookie is set and why we expect it to be empty here.